tests/integration/terraform/versions.tf

```diff
@@ -2,9 +2,8 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.60.0, <= 3.69.0"
```

Comment:
I ended up removing this provider version requirement because the API gateway tf module we're using forces us on to 3.10.0 anyway
https://registry.terraform.io/modules/clouddrove/api-gateway/aws/latest?tab=dependencies
I am not wild about that. Not sure what we're trying to get out of that module exactly but it might be worth looking at an alternative (e.g. writing our own API gateway tf template).